Networking/Networking/Remote/JetpackConnectionRemote.swift

```diff
@@ -13,6 +13,34 @@ public final class JetpackConnectionRemote: Remote {
         super.init(network: network)
     }
 
+    /// Retrieves the information about Jetpack the plugin for the current site.
+    ///
+    public func retrieveJetpackPluginDetails(completion: @escaping (Result<SitePlugin, Error>) -> Void) {
+        let path = String(format: "%@/%@", Path.plugins, Constants.jetpackPluginName)
+        let request = WordPressOrgRequest(baseURL: siteURL, method: .get, path: path)
+        let mapper = SitePluginMapper(withDataEnvelope: false)
+        enqueue(request, mapper: mapper, completion: completion)
+    }
+
+    /// Installs Jetpack the plugin to the current site.
+    ///
+    public func installJetpackPlugin(completion: @escaping (Result<SitePlugin, Error>) -> Void) {
+        let parameters: [String: Any] = [Field.slug.rawValue: Constants.jetpackPluginSlug]
+        let request = WordPressOrgRequest(baseURL: siteURL, method: .post, path: Path.plugins, parameters: parameters)
+        let mapper = SitePluginMapper(withDataEnvelope: false)
+        enqueue(request, mapper: mapper, completion: completion)
+    }
+
+    /// Activates Jetpack the plugin to the current site
+    ///
+    public func activateJetpackPlugin(completion: @escaping (Result<SitePlugin, Error>) -> Void) {
+        let path = String(format: "%@/%@", Path.plugins, Constants.jetpackPluginName)
```

Comment:
nit: similar to above
```suggestion
        let path = "\(Path.plugins)/\(Constants.jetpackPluginName)"
```

Reply:
Also fixed in 7cb48953d255e92cf52db2da51d3ec8c20aa354a.